synthetics/test_schema.go

```diff
@@ -302,30 +309,33 @@ func makeTestHealthSettingsSchema(mode schemaMode) *schema.Schema {
 }
 
 func makeTestMonitoringSettingsSchema(mode schemaMode) *schema.Schema {
-	return makeNestedObjectSchema(mode, map[string]*schema.Schema{
+	return makeNestedObjectSchemaRequired(mode, map[string]*schema.Schema{
 		"activation_grace_period": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
```

Comment:
I think it is requires on create, but have not tested it. Have you?

Reply:
Yes, for hostname type it is not required.